src/domain/app/BrowserApp.tsx

```diff
@@ -24,11 +24,11 @@ const BrowserApp: React.FC = () => {
   return (
     <BrowserRouter>
       <ApolloProvider client={apolloClient}>
-        <ApolloHooksProvider client={apolloClient}>
-          <MatomoProvider value={instance}>
-            <App />
-          </MatomoProvider>
-        </ApolloHooksProvider>
+        {/* <ApolloHooksProvider client={apolloClient}> */}
```

Comment:
Is there a reason why these are commented out?

Reply:
Deleted, forgot to push